client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -29,6 +30,16 @@ export default function Dashboard(): JSX.Element {
 
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
+      <Navbar
+        logOut="me"
+        profile="hi"
+        mySitter="My Sitters"
+        becomeSitter="BECOME A SITTER"
+        messages="Messages"
```

Comment:
I believe these are routes passed down to the navbar, in that case I would recommend using an [enum-like object](https://www.sohamkamani.com/javascript/enums/#enumerations-with-objects) to define routes that can be used project-wide.

Reply:
followed the standard Enumerations with objects